installation/sources/build-and-install.md

```diff
@@ -73,144 +73,213 @@ it's likely you may need root privileges so you can try to prefixing the command
 
 ## Build Options
 
-Fluent Bit provides certain options to CMake that can be enabled or disabled when configuring, please refer to the following tables under the _General Options_, _Development Options_, Input Plugins _and \_Output Plugins_ sections.
+Fluent Bit provides certain options to CMake that can be enabled or disabled when configuring, please refer to the following tables under the _General Options_, _Development Options_, Input Plugins _and _Output Plugins_ sections.
```

Comment:
Since you're editing this line, it'd be good to get some style changes in place, both for line length, and to hew closer to our updated general style recommendations for Fluent Bit docs.

```suggestion
Fluent Bit provides options to CMake that you can enable or disable when configuring.
Refer to the following tables in the **General Options**, **Development Options**,
**Input Plugins** and **Output Plugins** sections.
```